docs/functional-components/static-assets.md

```diff
@@ -0,0 +1,26 @@
+## Static assets
+
+## Recommended packages
+
+* [express-static](https://expressjs.com/en/starter/static-files.html). 
+Express static is part of the express.js package that allows developers to expose static middlewares.
+
+## Guidance
+
+For serving static resources we recomend using `express.static` middleware as it has been widely used and tested in production.
+When using express.static() method, we can server static resources directly by specifying the folder name where we have stored our static resources.
+
+Documentation for the middleware can be found in:
+https://expressjs.com/en/starter/static-files.html
```

Comment:
Additional guidance for why or why not to serve assets out of application

## Serving assets out of application
* Makes application docker images self serving of dependencies (no need to deploy/sync assets to S3 to have application running)
	* No need for dedicate step to sync assets (less devops headache)
	* No need for application code to hard code domain path for s.w-x.co
	* Allows relative path for assets for ease of portability
* DeployReviews could go straight to production (since docker images are self serving, these are not dependent on external resources like S3)
* Misconception: Assets do not bloat the docker image as they are small in comparison to poorly written docker images which do not appropriately remove unused code. Eg docker node-alpine image vs node.)
* Assets will already be fronted by Akamai CDN and incur less of a hit to origin.
* Sabotages h2 push (not able to utilize h2 due to separate domain)

## Serving assets out of S3
* Has benefit of not incurring application origin hit cost
	* Although static assets would still be fronted by Akamai CDN
* Has issues syncing assets from environment to environment
* COS migration is harder as application is dependent on specific buckets/domain names for where assets will live.
	* Deploying to specific COS buckets means specific secret and access keys per bucket
	* Deploying to specific COS buckets also means associated akamai rules per bucket

Reply:
@roastlechon knowing how much effort you have put I would not feel good to apply this change myself. I think it will be the best if you take this over and apply this change directly. Simply because this is exactly what we talked about and I would not say it better.

Reply:
i can pull your branch and create new PR if that would be okay by you. I can reference existing comments/feedback in the new PR, while you focus more on the other PRs (graphql)

Reply:
Anything you will like. Really I haven't done much and amount not knowledge you have brought here is amazing. Feel free to create new PR, close this one - anything that will minimize hassle on your side